### DISCUSSION TITLE
Fault diagnosis with machine learning

### DISCUSSION THREAD

**Discussion by morgand-sql** [mrdbourke/pytorch-deep-learning#878] [General]
I am seeking assistance in developing a research topic focused on the application of machine learning in fault diagnosis. I would greatly appreciate any help or recommendations you can provide. @mrdbourke, @shashin

**Comment by LuluW8071** [comment]
@moranaudbxhb You can send your research topic details through mail :)

## [Send Email](mailto:shashinmaharjan@gmail.com)

**Comment by iibrahemali** [comment]
Explore the effectiveness of deep learning techniques, such as convolutional neural networks (CNNs) or recurrent neural networks (RNNs), in detecting anomalies and faults in complex industrial systems, such as manufacturing processes or power grids.
(DL for Anomaly Detection in Industrial Systems)
